warehouse/query-core/src/test/java/datawave/query/jexl/nodes/QueryPropertyMarkerTest.java

```diff
@@ -29,11 +30,11 @@ public void testFindInstance() throws ParseException {
         assertEmptyInstance(findInstance("((_Bounded_ = true) && (FOO > 1 && FOO < 2)) && ((_EvalOnly_ = true) && (FOO == 1 && BAR == 2))"));
         
         // Test unwrapped marker and unwrapped sources
-        assertInstance(findInstance("(_Delayed_ = true) && FOO == 1 && BAR == 2"), ASTDelayedPredicate.class, "(BAR == 2 && FOO == 1)");
+        assertInstance(findInstance("(_Delayed_ = true) && FOO == 1 && BAR == 2"), ASTDelayedPredicate.class, "(FOO == 1 && BAR == 2)");
         // Test unwrapped marker and wrapped sources
         assertInstance(findInstance("(_Delayed_ = true) && (FOO == 1 && BAR == 2)"), ASTDelayedPredicate.class, "FOO == 1 && BAR == 2");
         // Test wrapped marker and unwrapped sources
-        assertInstance(findInstance("((_Delayed_ = true) && FOO == 1 && BAR == 2)"), ASTDelayedPredicate.class, "(BAR == 2 && FOO == 1)");
+        assertInstance(findInstance("((_Delayed_ = true) && FOO == 1 && BAR == 2)"), ASTDelayedPredicate.class, "(FOO == 1 && BAR == 2)");
```

Comment:
In the spirit of "you broke it you bought" and "you touched it last", do you mind putting single quotes around the values in these queries? e.g., `FOO == '1'`

(sorry)

Reply:
Haha, no problem!